citrination_client/client.py

```diff
@@ -67,6 +67,32 @@ def upload_file(self, file_path, data_set_id):
         else:
             return None
 
+    def show_latest_dataset_files(self, dataset_id):
+        return self._get_content_from_url(self.api_url + '/data_sets/' + str(dataset_id) + '/latest')
+
+    def show_dataset_files(self, dataset_id):
```

Comment:
What is the difference between this and the previous method, show_latest_dataset_files?